src/main/java/org/fundacionjala/coding/ketty/CamelCase.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.coding.ketty;
+
+import java.util.StringJoiner;
+
+/**
+ * @author ketty
+ * class CamlCase of a string.
+ */
+
+public class CamelCase {
+    /**
+     * @param phrase it is string :
+     * @return count.
+     */
+    public String camelCase(final String phrase) {
+        StringJoiner union = new StringJoiner("");
```

Comment:
For this scenario is not necessary to use a StringJoiner, it is enough with a StringBuilder